docs/content/grafana/README.md

```diff
@@ -21,12 +40,54 @@ To add your dashboards to Grafana, you will need to create a `GrafanaDashboard`
 
-You can import these dashboards in Grafana either by pasting the dashboard JSON text directly into the text area or directly uploading the JSON file. Make sure to add/connect your [data sources][8] to Grafana and pick what data source you want the dashboard to use.
+You can import these dashboards in Grafana either by pasting the dashboard `JSON` text directly into the text area or directly uploading the `JSON` file. Make sure to add/connect your [data sources][8] (official Grafana data source documentation) to Grafana and pick what data source you want the dashboard to use.
 
 For more details on importing/exporting dashboards, you can refer to the Grafana documentation [here][9].
 
+## Creating Your First Dashboard
+
+If you've gained access to the Operate First Grafana instance, you should see the dashboard pictured above.
+
+On the left side bar there is a "+" button. Hover over it and click "Dashboard". This will create a new dashboard for you, where Grafana should redirect you immediately to.
+
+Once in your new dashboard, there are many options to create new visualizations (the usage of this is beyond the scope of this document). Once you've made a dashboard, you should save it. The save icon is located in the upper right-hand corner of the dashboard, and clicking it will prompt you for a name and a folder. Upon saving, your dashboard will be in the Grafana instance.
+
+## Backing Up and Exporting Your Dashboard
+
+Now that you saved your dashboard to our Grafana instance you will now have to backup your dashboard locally, what does this mean? The Grafana instance/operator on OpenShift can be restarted (whether it be for updates or a new dashboard added via a pull request). Whenever a restart happens all instance saved dashboards will be deleted (the dashboard you just created and saved). This means that any dashboard you save is ephemeral unless you add the dashboard to the apps repository found [here](https://github.com/operate-first/apps).
+
+Adding your dashboard can be done by following this guide from the [GitOps docs](https://www.operate-first.cloud/apps/content/grafana/add_grafana_dashboard.html). The process of adding a dashboard looks like this:
+- Export the dashboard via `JSON`
+  - Click the share icon on (left side of top bar), click on "Export" and then click "Save to file". You will now have a `JSON` file of your dashboard
+
+ Here you would want to follow the guide linked above, in your `<dashboard-name>.yaml` file you can paste the `JSON` like such
+```yaml
+---
+apiVersion: integreatly.org/v1alpha1
+kind: GrafanaDashboard
+metadata:
+  labels:
+    app: grafana
+  name: namespace-storage-overview
+spec:
+  customFolderName: Cluster Management
+  plugins:
+    - name: grafana-piechart-panel
+      version: 1.6.2
+  json: |
+    {
+      "annotations": {
+        "list": [
+          {
+  .............
+```
```

Comment:
This is fine: 

> Adding your dashboard can be done by following this guide from the [GitOps docs](https://www.operate-first.cloud/apps/content/grafana/add_grafana_dashboard.html). 

But the part that follows here: 

> The process of adding a dashboard looks like this....

seems like it's just repeating the instructions [here](https://www.operate-first.cloud/apps/content/grafana/add_grafana_dashboard.html#backing-up-your-dashboard). So I don't think we need to repeat this here. But you can certainly improve the section I've linked if you feel more instructions are needed there.